src/main/java/uk/nhs/hee/tis/common/upload/service/AwsStorageService.java

```diff
@@ -133,15 +136,8 @@ public String getData(StorageDto storageDto) {
         String sortDirection = sortEntry[1];
 
         Function<? super FileSummaryDto, String> extractor = o -> getStringProperty(o, sortKey);
-        Comparator<? super String> comparator;
-        switch (sortDirection) {
-          case "desc":
-            comparator = Comparator.reverseOrder();
-            break;
-          case "asc":
-          default:
-            comparator = Comparator.naturalOrder();
-        }
+        Comparator<? super String> comparator =
+            "desc".equals(sortDirection) ? Comparator.reverseOrder() : Comparator.naturalOrder();
```

Comment:
Would prefer `Objects.equals(sortDirection, "desc")` here, but major issue it is not.